### DISCUSSION TITLE
Chatgpt clone php

### DISCUSSION THREAD

**Discussion by alexsky177** [alexsky177/chat#2] [General]
ChatGpt php clone beta.
 All function is work ok.
 Frondpage is same as this project https://github.com/xtekky/chatgpt-clone. 
No have some functions: change model, change tenperature, stop generating unswer, can not open options new chat and delete chat in mobile version. 
Easily installed.

Copy all files. 
Change Open AI API key in chat.php. 
ALL is finished. 
Can use. 
Go to index.html any browser or mobile device

<hr /><em>This discussion was created from the release <a href='https://github.com/alexsky177/chatgpt/releases/tag/'>Chatgpt clone php</a>.</em>

**Comment by ZachTheJack** [comment]
Just curious, why did you get rid of the change model and web browse features from https://github.com/xtekky/chatgpt-clone ?

I'm interested in adding them back, going to look into that if its not something you're already looking at doing?

Great work though, uploaded to my server and it's working like a charm. Added a password on it too so I don't have random people using my API haha

**Comment by alexsky177** [comment]
Hi
I do not have a subscription to gpt-4, and there is practically no
difference between gpt-3.5-turbo and gpt-3.5-turbo-0301.
I can add this function later.
Also i made new version on JavaSrcipt only. No need any php or pyton.
https://www.builder.io/blog/stream-ai-javascript
I will be add it soon.

ср, 24 мая 2023 г. в 06:47, Zyler Kade ***@***.***>:
